src/test/java/pl/edu/icm/pl/mxrdr/extension/workflow/XdsSupplyMissingValuesStepTest.java

```diff
@@ -0,0 +1,131 @@
+package pl.edu.icm.pl.mxrdr.extension.workflow;
+
+import edu.harvard.iq.dataverse.dataset.datasetversion.DatasetVersionServiceBean;
+import edu.harvard.iq.dataverse.persistence.dataset.DatasetField;
+import edu.harvard.iq.dataverse.persistence.dataset.DatasetFieldType;
+import edu.harvard.iq.dataverse.persistence.dataset.DatasetVersion;
+import edu.harvard.iq.dataverse.workflow.execution.WorkflowExecutionContext;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import pl.edu.icm.pl.mxrdr.extension.importer.MxrdrMetadataField;
+
+import javax.enterprise.inject.Vetoed;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Predicate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class XdsSupplyMissingValuesStepTest {
+
+    private static final String TEST_PATH_PREFIX = "xds/xsmv-test-";
+    private static final String VALUE = "1234.5";
+
+    private Path workDir;
+
+    private WorkflowExecutionContext context = Mockito.mock(WorkflowExecutionContext.class, Mockito.RETURNS_DEEP_STUBS);
+
+    @BeforeEach
+    public void setUp() throws IOException {
+        workDir = Files.createTempDirectory("xds-test-temp");
+        String inputFileSourcePath = getFileFromResources(XdsAdjustResultStep.XDS_INP);
```

Comment:
Explicit `getFileFromResources("xds/xsmv-test-" + XdsAdjustResultStep.XDS_INP)` seems more readable instead of implicitly relying on a side effect of the function using a constant defined elsewhere?